src/main/kotlin/io/getunleash/UnleashClient.kt

```diff
@@ -88,7 +88,10 @@ class UnleashClient(
     }
 
     override fun getVariant(toggleName: String): Variant {
-        val variant = refreshPolicy.readToggleCache()[toggleName]?.variant ?: disabledVariant
+        val variant =
+                if (isEnabled(toggleName))
+                        refreshPolicy.readToggleCache()[toggleName]?.variant ?: disabledVariant
+                else disabledVariant
```

Comment:
```suggestion
                if (isEnabled(toggleName)) {
                        refreshPolicy.readToggleCache()[toggleName]?.variant ?: disabledVariant
} else {
  disabledVariant
}
```
I know it's allowed to not do it, but if/else should always have braces. I was pretty sure we had a lint for this.

Reply:
Also, I see I broke the formatting here. The suggestion box didn't show properly

Reply:
Resolved in 126fbd07217db581e946a463988459fd4a1baaa8